front/src/styles/scss/applications/operationalStudies/components/_AddAndEditProjectModal.scss

```diff
@@ -0,0 +1,118 @@
+.project-edition-modal {
+  .project-edition-modal-picture {
+    position: absolute;
+    overflow: hidden;
+    border-radius: 4px;
+    max-height: 30vh;
+    width: 100%;
+    height: 100%;
+    img {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      border-radius: 4px;
+      color: var(--coolgray7);
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+      transition: opacity 0.2s;
+      &:hover {
+        opacity: 0.75;
+      }
+    }
+    .project-edition-modal-picture-placeholder {
+      position: relative;
+      font-size: 5rem;
+      background-color: var(--coolgray1);
+      color: var(--coolgray7);
+      border-radius: 4px;
+      width: calc(100% - 1rem);
+      height: 100%;
+      transition: 0.2s;
+      label {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        width: 100%;
+        height: 100%;
+        cursor: pointer;
+        .project-edition-modal-picture-placeholder-text {
+          font-weight: 500;
+          font-size: 1rem;
+          color: var(--coolgray9);
+        }
+      }
+      &:hover {
+        background-color: var(--coolgray3);
+        color: var(--coolgray9);
+      }
+      .project-edition-modal-picture-placeholder-buttons {
+        display: flex;
+        align-items: center;
+        gap: 0.5rem;
+        position: absolute;
+        z-index: 1;
+        right: 0.5rem;
+        bottom: 0.5rem;
+        button {
+          display: flex;
+          align-items: center;
+          background-color: var(--white);
+          border: none;
+          color: var(--coolgray11);
+          font-size: 1.25rem;
+          border-radius: 4px;
+          padding: 0.5rem;
+          transition: 0.2s;
+          &:hover {
+            background-color: var(--coolgray11);
+            color: var(--coolgray1);
+          }
+          &.remove {
+            color: var(--red);
+            &:hover {
+              background-color: var(--red);
+              color: var(--white);
+            }
+          }
+        }
+      }
+    }
+  }
+  .project-edition-modal-title {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    width: 100%;
+    img {
+      height: 2rem;
+    }
+  }
+  .project-edition-modal-name {
+    height: 7.5rem;
+  }
+  .project-edition-modal-description {
+    textarea {
+      min-height: auto;
+    }
+  }
+  .project-edition-modal-objectives {
+    textarea {
+      height: 20vh;
+      padding: 0.5rem;
+      font-size: 0.7rem;
+      font-family: monospace;
+    }
+  }
+  .project-edition-modal-objectives-md {
+    height: 100%;
+    .project-edition-modal-objectives-md-render {
+      overflow: auto;
+      height: 20vh;
+      padding: 0.5rem;
+      border-radius: 4px;
+      border: 1px solid var(--coolgray3);
+    }
+  }
+}
```

Comment:
missing new line

Reply:
Corrected all end lines in scss (activated option in vscode)